datahub-web-react/src/app/entity/shared/tabs/Dataset/Validations/DatasetAssertionDescription.tsx

```diff
@@ -181,7 +181,7 @@ const getAggregationText = (
         case DatasetAssertionScope.DatasetRows:
             return getRowsAggregationText(aggregation);
         case DatasetAssertionScope.DatasetColumn:
-            return getColumnAggregationText(aggregation, fields?.length === 1 ? fields[0] : undefined);
+            return getColumnAggregationText(aggregation, fields?.length === 1 ? fields[0] : fields.join(", "));
```

Comment:
hm i'm not sure this would fix the whole problem here since `getColumnAggregationText` expects a single column. would you be able to look into how we could support multiple columns in this function or even create a new function for multi-column use cases?